app/uk/gov/hmrc/apiplatform/modules/mfa/controllers/profile/ProtectAccount.scala

```diff
@@ -38,35 +39,33 @@ import views.html.protectaccount._
 
 import scala.concurrent.{ExecutionContext, Future}
 import uk.gov.hmrc.thirdpartydeveloperfrontend.controllers.LoggedInController
-import uk.gov.hmrc.thirdpartydeveloperfrontend.controllers.Remove2SVConfirmForm
 import uk.gov.hmrc.thirdpartydeveloperfrontend.controllers.FormKeys
 
 @Singleton
 class ProtectAccount @Inject()(
-  val thirdPartyDeveloperConnector: ThirdPartyDeveloperConnector,
-  val thirdPartyDeveloperMfaConnector: ThirdPartyDeveloperMfaConnector,
-  val otpAuthUri: OtpAuthUri,
-  val mfaService: MFAService,
-  val sessionService: SessionService,
-  mcc: MessagesControllerComponents,
-  val errorHandler: ErrorHandler,
-  val mfaMandateService: MfaMandateService,
-  val cookieSigner : CookieSigner,
-  protectAccountSetupView: ProtectAccountSetupView,
-  protectedAccountView: ProtectedAccountView,
-  protectAccountView: ProtectAccountView,
-  protectAccountAccessCodeView: ProtectAccountAccessCodeView,
-  protectAccountCompletedView: ProtectAccountCompletedView,
-  protectAccountRemovalConfirmationView: ProtectAccountRemovalConfirmationView,
-  protectAccountRemovalAccessCodeView: ProtectAccountRemovalAccessCodeView,
-  protectAccountRemovalCompleteView: ProtectAccountRemovalCompleteView
+                                val thirdPartyDeveloperConnector: ThirdPartyDeveloperConnector,
```

Comment:
Looks like formatting changes that should not be here.  I don't believe the formatter does this style so probably an IDE issue.

Reply:
Done